### PR TITLE
Add CH573 and D1x and BPI-F3 and Duo256M

### PR DESCRIPTION
**Pull request by wychlw**

### CODE REVIEW

File: Duo256m/FreeRTOS/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# BuildRoot Milk-V Duo 256M 测试报告
+
+## 测试环境
+
+### 操作系统信息
+
+- 构建系统：Ubuntu 22.04.3 LTS x86_64
+- 系统版本：Duo-V1.0.9
```
Comment (KevinMX) [inline Duo256m/FreeRTOS/README.md]: Should be 1.10 now?

File: BPI-F3/Bianbu copy/README.md
Comment (KevinMX) [inline BPI-F3/Bianbu copy/README.md:1]: Bianbu copy?

**Review by KevinMX — CHANGES_REQUESTED** [review]

File: BPI-F3/Bianbu copy/README.md
Comment (wychlw) [reply BPI-F3/Bianbu copy/README.md:1]: My bad

File: Duo256m/FreeRTOS/README.md (hunk 1 above)
Comment (wychlw) [reply Duo256m/FreeRTOS/README.md]: Well... Then I delete these as adding them after test makes more sense?

File: mangopi_mq_pro/Ubuntu/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# Armbian MangoPi MQ Pro 测试报告
```
Comment (KevinMX) [inline mangopi_mq_pro/Ubuntu/README.md]: Armbian -> Ubuntu?

**Review by KevinMX — CHANGES_REQUESTED** [review]

File: mangopi_mq_pro/Ubuntu/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+## 测试环境
+
+### 操作系统信息
+
+- 下载链接：https://cdimage.ubuntu.com/releases/23.10/release/ubuntu-23.10-preinstalled-server-riscv64+nezha.img.xz
```
Comment (KevinMX) [inline mangopi_mq_pro/Ubuntu/README.md:7]: ~~Not sure if nezha's image will work, there might be some differences in dtb~~
Never mind, this should be fine

**Review by KevinMX — CHANGES_REQUESTED** [review]

**Review by KevinMX — APPROVED** [review]

**Review by KevinMX — APPROVED** [review]

**Pull request merged**